spec/core/filtering/simple_rule_spec.rb

```diff
@@ -39,7 +39,7 @@
 
   shared_examples_for 'a match' do
     it 'matches' do
-      expect(subject.match?(document)).to be
+      expect(subject.match?(document)).to be_truthy
```

Comment:
:P

Reply:
yeah IDK what it's supposed to mean but it failed for me 😢